bindings/python/test/test_bson.py

```diff
@@ -18,12 +18,12 @@
 from pymongoarrow.context import PyMongoArrowContext
 from pymongoarrow.lib import process_bson_stream
 from pymongoarrow.schema import Schema
-from pymongoarrow.types import int32, int64
+from pymongoarrow.types import int64, ObjectId
 
 
 class TestBsonToArrowConversionBase(TestCase):
     def setUp(self):
-        self.schema = Schema({'_id': int32(),
+        self.schema = Schema({'_id': ObjectId,
```

Comment:
Are we now loosing test coverage for int32?

Reply:
All of the arrow types are checked in `test_pandas.py`